x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -73,6 +75,9 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
   constructor(initializerContext: PluginInitializerContext) {
     this.kibanaVersion = initializerContext.env.packageInfo.version;
   }
+  private detectionsUpdater$ = new BehaviorSubject<AppUpdater>(() => ({}));
```

Comment:
Would you explain this (in a comment perhaps?) Why do these BehaviorSubject's take such functions?

Reply:
it's an initial value, not needed in this case, so changing to just Subject